Resources/Locale/en-US/stack/stacks.ftl

```diff
@@ -228,6 +228,7 @@ stack-mowed-astro-grass-floor = mowed astro-grass floor
 stack-jungle-astro-grass-floor = jungle astro-grass floor
 stack-astro-ice-floor = astro-ice floor
 stack-astro-snow-floor = astro-snow floor
+stack-mowed-astro-snow-floor = mowed astro-snow floor
```

Comment:
Move to DeltaV namespaced file (it can be the same one as `tiles-mowed-astro-snow`)